taichi/codegen/codegen_opengl.cpp

```diff
@@ -832,6 +832,14 @@ void OpenglCodeGen::lower() {  // {{{
 #endif
 }  // }}}
 
+FunctionType fuck_cpp(std::unique_ptr<CompiledProgram> p)
```

Comment:
I have this feeling from time to time. But let's not commit it to public repo...